enrollment_form/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# syntax=docker/dockerfile:1
+FROM node:20-alpine AS base
+
+WORKDIR /app
+
+ENV NEXT_PUBLIC_BASE_PATH /enrollment
+
+COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
+RUN \
+  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
+  elif [ -f package-lock.json ]; then npm ci; \
+  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i --frozen-lockfile; \
+  else echo "Lockfile not found." && exit 1; \
+  fi
+
+COPY . .
+RUN yarn build
+
+FROM node:20-alpine AS production
+WORKDIR /app
+ENV NODE_ENV production
+ENV NEXT_PUBLIC_BASE_PATH /enrollment
+
+# Copy the built application from the previous stage
+COPY --from=base /app/.next ./.next
+COPY --from=base /app/node_modules ./node_modules
```

Comment:
Because it currently does not use the `standalone` build (running into issues with the `basePath`). During the `yarn build` without standalone, a `.next` dir is created with just the compiled application code, static assets, and server-rendered pages. The `node_modules` is still used for the runtime dependencies.